polyfill/package.json

```diff
@@ -83,6 +83,7 @@
     "full-icu": "^1.4.0",
     "nyc": "^15.1.0",
     "rollup": "^2.68.0",
+    "rollup-plugin-node-polyfills": "^0.2.1",
```

Comment:
What is the plugin needed for? Does CopyDataProperties depend on it?

Reply:
I think call-bind was breaking without it.

Reply:
oof, does rollup break things just like webpack 5, by not automatically polyfilling node builtins?

Reply:
Yeah, that seems about right. You should be able to observe the problem if you checkout this PR, remove the rollup-plugin-node-polyfills dependency, and run `npm install && npm run build` (maybe followed by `npm test && npm run test262`; I don't remember if the issue manifested at build time).